cache-config/t3cutil/t3cutil.go

```diff
@@ -228,3 +232,14 @@ func UserAgentStr(appName string, versionNum string, gitRevision string) string
 	}
 	return appName + "/" + versionNum + ".." + gitRevision
 }
+
+// InstallDirectory returns the location of the executable directory.
+// If error, returns "/usr/bin" as default.
+func InstallDirectory() string {
+	installdir, err := osext.ExecutableFolder()
```

Comment:
ATC is on Go >1.18 which has `os.Executable()`. Can you change this to `os.Executable`, and not use/vendor `osext`?